Library/Homebrew/token_auditor.rb

```diff
@@ -0,0 +1,40 @@
+# typed: true
+# frozen_string_literal: true
+
+module Homebrew
+  class TokenAuditor
```

Comment:
```suggestion
  class NameAuditor
```

Thoughts? Token is cask-specific so I think maybe "name" is clearer in this case. NBD though either way

Reply:
True, but also `token` has _no_ meaning for formulae. When I'm speaking about what to call a formula or cask, I use the word "name" and mean `Formula#name` and `Cask#token`.

Reply:
```suggestion
  class FormulaNameCaskTokenAuditor
```
or something.

I agree `TokenAuditor` doesn't make sense if it can operate on Formula because they don't have tokens.